scraper/settings.py

```diff
@@ -80,7 +80,7 @@
 DATABASE = {
     'drivername': 'postgres',
     'host': 'localhost',
-    'port': '54321',
+    'port': '5432',
```

Comment:
Oops, I had it running on the default port, but I'll change this back to what it was

Reply:
Let's can keep it as 5432 since that is the default port.  I must have updated my settings b/c I was running another instance of postgres locally